packages/main/src/TableUtils.ts

```diff
@@ -20,7 +20,11 @@ const findRowInPath = (composedPath: Array<EventTarget>) => {
 const findVerticalScrollContainer = (element: HTMLElement): HTMLElement => {
 	while (element) {
 		const { overflowY } = window.getComputedStyle(element);
-		if (overflowY === "auto" || overflowY === "scroll") {
+		if (overflowY === "auto") {
+			if (element.scrollHeight > element.clientHeight) {
```

Comment:
the rest of the change is OK but I am not sure about this one. if a parent has an overflow value rather than default "visible" then it should be responsible for the scrolling, it might be not scrollable at the moment but it can be scrollable suddenly.
Just wanted to double check before +2